app/service.js

```diff
@@ -0,0 +1,184 @@
+angular.module('resume')
+    .service('ResumeService', function () {
+
+	var imgRoot = 'asset/img/';
+
+    socialLinks = [{
+            url: 'https://www.facebook.com/radkov.sn',
+			title: "Facebook",
+			img: imgRoot + 'social/facebook_icon.png'
+		    }, {
+            url: 'https://www.linkedin.com/in/radkovsn',
+			title: "LikedIn",
+			img: imgRoot + 'social/linkedin_icon.png'
+			}, {
+            url: 'https://github.com/sergeyradkov',
+			title: "GitHub",
+			img: imgRoot + 'social/github_icon.png'
+			}, {
+            url: 'https://www.instagram.com/sergeriic/',
+			title: "Instagram",
+			img: imgRoot + 'social/instagram_info.png'
+	 }];
+     
+     this.getSocialLinks = function (){
+         return socialLinks;
+     }
```

Comment:
Missing semicolon here an in many other places. Please, use any static code analysis tool to check your code and improve quality. I'd recommend installing [JsHint](http://jshint.com/): `npm install -g jshint`
Usage: `jshint app`
```
app/app.js: line 1, col 62, Missing semicolon.

app/scroll.js: line 4, col 5, Bad line breaking before '&&'.
app/scroll.js: line 7, col 7, Bad line breaking before '||'.

app/service.js: line 26, col 7, Missing semicolon.
app/service.js: line 111, col 15, Unnecessary semicolon.
app/service.js: line 112, col 11, Unnecessary semicolon.
app/service.js: line 158, col 15, Unnecessary semicolon.
app/service.js: line 160, col 11, Unnecessary semicolon.
app/service.js: line 162, col 7, Unnecessary semicolon.
app/service.js: line 173, col 7, Missing semicolon.
app/service.js: line 183, col 7, Missing semicolon.

11 errors
```